openpype/pipeline/colorspace.py

```diff
@@ -0,0 +1,464 @@
+    """Get all colorspace data
+
+    Wrapper function for aggregating all names and its families.
+    Families can be used for building menu and submenus in gui.
+
+    Args:
+        config_path (str): path leading to config.ocio file
+
+    Returns:
+        dict: colorspace and family in couple
+    """
+    if sys.version_info[0] == 2:
+        return get_colorspace_data_subprocess(config_path)
+
+    from ..scripts.ocio_wrapper import get_colorspace_data
+    return get_colorspace_data(config_path)
+
+
+def get_colorspace_data_subprocess(config_path):
+    """Get colorspace data via subprocess
+
+    Wrapper for Python 2 hosts.
+
+    Args:
+        config_path (str): path leading to config.ocio file
+
+    Returns:
+        dict: colorspace and family in couple
+    """
+    with _make_temp_json_file() as tmp_json_path:
+        # Prepare subprocess arguments
+        args = [
+            "run", get_ocio_config_script_path(),
+            "config", "get_colorspace",
+            "--in_path", config_path,
+            "--out_path", tmp_json_path
+
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
+
+        process_kwargs = {
+            "logger": log,
+            "env": {}
+        }
+
+        run_openpype_process(*args, **process_kwargs)
+
+        # return all colorspaces
+        return_json_data = open(tmp_json_path).read()
+        return json.loads(return_json_data)
+
+
+def get_ocio_config_views(config_path):
+    """Get all viewer data
+
+    Wrapper function for aggregating all display and related viewers.
+    Key can be used for building gui menu with submenus.
+
+    Args:
+        config_path (str): path leading to config.ocio file
+
+    Returns:
+        dict: `display/viewer` and viewer data
+    """
+    if sys.version_info[0] == 2:
+        return get_views_data_subprocess(config_path)
+
+    from ..scripts.ocio_wrapper import get_views_data
+    return get_views_data(config_path)
+
+
+def get_views_data_subprocess(config_path):
+    """Get viewers data via subprocess
+
+    Wrapper for Python 2 hosts.
+
+    Args:
+        config_path (str): path leading to config.ocio file
+
+    Returns:
+        dict: `display/viewer` and viewer data
+    """
+    with _make_temp_json_file() as tmp_json_path:
+        # Prepare subprocess arguments
+        args = [
+            "run", get_ocio_config_script_path(),
+            "config", "get_views",
+            "--in_path", config_path,
+            "--out_path", tmp_json_path
+
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
+
+        process_kwargs = {
+            "logger": log,
+            "env": {}
+        }
+
+        run_openpype_process(*args, **process_kwargs)
+
+        # return all colorspaces
+        return_json_data = open(tmp_json_path).read()
+        return json.loads(return_json_data)
+
+
+def get_imageio_config(
+    project_name, host_name,
+    project_settings=None,
+    anatomy_data=None,
```

Comment:
Should this be renamed to `template_data` maybe? Since the API functions to retrieve the data are also called: `get_template_data_from_session`, `get_template_data_with_names` and `get_template_data`?

Reply:
Yeah, it seems like you are right. We were previously using anatomy data and if you search trough openpype codebase this is always referred as `anatomy_data`. I will keep it this way.